packages/examples/simple/legacy.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
I found these useful for a manual end-to-end test of the simple example, so I'm checking them in.